otherlibs/unix/unix.mli

```diff
@@ -881,6 +881,9 @@ val open_process_args_in : string -> string array -> in_channel
    The standard output of the command is redirected to a pipe, which can be read
    via the returned input channel.
 
+   The program is looked up in the [PATH] (this behaviour changed in 4.12;
+   previously the program was interpreted relative to the current directory).
```

Comment:
Elsewhere in unix.mli we write "The executable file is searched in the path".  (4 occurrences.)  

Also, you first talk about "the command" and then "the program".  For me, "the command" is a shell command line, as used in `open_process_in` for instance.  "The program" is better.

Also, "the program was interpreted relative to the current directory" is still true: the path is not searched for programs like `./foo` or `foo/bar`.